calico-enterprise/getting-started/install-on-clusters/rke2.mdx

```diff
@@ -130,6 +131,58 @@ Monitor progress with the following command:
 watch kubectl get tigerastatus
 ```
 
+</TabItem>
+
+<TabItem label="Rancher UI" value="UI-1">
+
+Unlike the Rancher CLI, the Rancher UI does not provide an option to set the RKE2 CNI value to `none`, which is required by RKE2 to install a non-default CNI like {{prodname}}. To install {{prodname}} using Rancher UI, you must first provision an RKE2 cluster with Calico Open Source, then upgrade it to {{prodname}}.
+
+### Provision RKE2 cluster with Calico Open Source
+
+1. [Provision an RKE2 cluster using Calico as CNI and the default config option](https://ranchermanager.docs.rancher.com/pages-for-subheaders/launch-kubernetes-with-rancher)
+2. Validate the RKE2 cluster is set up and running.
+3. Update the Cluster Config by adding the following `HelmChartConfig` to the Additional Manifest section. (In the UI, navigate to **Edit Config**, **Add on config**, **Additional Manifest**.)
```

Comment:
This is the old step.

Reply:
Fixed.

Reply:
Not seeing the change.

Reply:
Actually on second thought, shouldn't the RancherUI section be removed from RKE2?

Reply:
Removed.